### ISSUE TITLE
Add React.memo support

### ISSUE THREAD

**Issue by Rycochet** [runreflect/webpack-react-component-name#7]
Having an issue where memoised components don't get an automatic component name so it creates an eslint error. Would be nice if it was also supported from here to simplify things.

**Comment by tmcneal** [comment]
It looks like we can automatically add `displayName` when the memoized component is assigned to a const:

```
import Button from "./Button";
import React from 'react';
const MemoizedButton = React.memo(Button);

export default MemoizedButton
```

but can not add it it's inlined like this:

```
import Button from "./Button";
import React from 'react';

export default React.memo(Button);

```

In the former case with my upcoming change we'll set a displayName matching the name of the const (e.g. MemoizedButton in that case).

Does this work for you?

**Comment by Rycochet** [comment]
We generally try not to use `export default` anywhere (it's generally a bad idea, though there are definitely exceptions) - so `export const MemoizedButton = React.memo(() => { ... });` is the general pattern we use - and looks like it'll be ok with your change? :-)

**Comment by tmcneal** [comment]
Yep I've confirmed that works - we'll have unit test coverage for that as well as the original variant I mentioned.  Should have a new release out soon!

**Comment by tmcneal** [comment]
Thanks again for the report. THis is fixed in `0.0.4`!
